src/SymfonyCache/CacheEvent.php

```diff
@@ -39,10 +38,10 @@ class CacheEvent extends Event
     private $response;
 
     /**
-     * @param HttpCache $kernel  The kernel raising with this event.
-     * @param Request   $request The request being processed.
+     * @param CacheManipulationInterface $kernel  The kernel raising with this event.
+     * @param Request                    $request The request being processed.
      */
-    public function __construct(HttpCache $kernel, Request $request)
+    public function __construct(CacheManipulationInterface $kernel, Request $request)
```

Comment:
Yes, as it’s a BC break I would prefer to postpone this to 2.0. Should we leave this PR open till then?